.env.example

```
@@ -1,5 +1,10 @@
 DATABASE_URL='postgresql://neondb_owner:npg_************************@ep-super-tree-awel6tvz-pooler.c-12.us-east-1.aws.neon.tech/neondb?sslmode=require&channel_binding=require'
 
+
+APP_MODE="prod"
+
+ENCRYPTION_KEY="---------------------------------------------"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)